### DISCUSSION TITLE
Step by step tutorial to implement testContainers .Net

### DISCUSSION THREAD

**Discussion by eliassal** [testcontainers/testcontainers-dotnet#892] [Q&A] [answered]
Hi I came across your tool this morning and would like to start testing it. Can you please let me know if there are some artiles/posts something like getting startedm thanks in advance

**Comment by HofmeisterAn** [comment, marked as answer]
Hello 👋 there are two beginner guides available. You can find them at the following links: [here](https://testcontainers.com/guides/introducing-testcontainers/) and [here](https://testcontainers.com/guides/getting-started-with-testcontainers-for-dotnet/). Additionally, you find further information in our [documentation](https://dotnet.testcontainers.org/). If you require any assistance, please feel free to reach out to us on [Slack](https://slack.testcontainers.org/).

**Comment by eliassal** [comment]
So many thanks
